galaxy_ng/app/tasks/publishing.py

```diff
@@ -52,12 +52,27 @@ def import_and_auto_approve(artifact_pk, **kwargs):
     Custom task to call pulp_ansible's import_collection() task
     then automatically approve collection version so no
     manual approval action needs to occur.
-
-    Approval currently is a collection version certification flag
-    Approval later will be moving from a staging to an approved repo
     """
-    import_collection(artifact_pk=artifact_pk, repository_pk=kwargs.get('repository_pk', None))
+    inbound_repository_pk = kwargs.get('repository_pk')
```

Comment:
I think the two tasks are pretty close to be able to be parameterized, especially in combo with https://github.com/ansible/galaxy_ng/pull/336/files

I think those two tasks could be one task with the inbound_repo as a kwarg if we wanted to.
At that point it is getting pretty close to pulpcore's add_or_remove task: https://github.com/pulp/pulpcore/blob/master/pulpcore/app/tasks/repository.py#L121

But let's get this pr and #336 merged and then refactor the tasks if needed.